x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-control-numbers.adoc

```diff
@@ -0,0 +1,28 @@
+= X12 EDI Connector 2.5 - Control Numbers - Mule 4
+
+Using the Write operation in the Anypoint X12 EDI Connector, you can generate control numbers on the generated X12 payload by:
```

Comment:
Same comments as for Line 3

Reply:
In fact, since these are the same texts, same comments as for the previous topic. 
Edit ends here.